cedar-policy-core/src/evaluator/err.rs

```diff
@@ -127,7 +127,8 @@ fn pretty_type_error(expected: &[Type], actual: &Type) -> String {
 
 #[derive(Debug, PartialEq, Clone, Error)]
 pub enum IntegerOverflowError {
-    #[error("integer overflow while attempting to {} the values {arg1} and {arg2}", match .op { BinaryOp::Add => "add", BinaryOp::Sub => "subtract", _ => "perform an operation on" })]
+    /// Overflow during a binary operation
+    #[error("integer overflow while attempting to {} {arg1} by {arg2}", match .op { BinaryOp::Add => "add", BinaryOp::Sub => "subtract", BinaryOp::Mul => "multiply", _ => "perform an operation on" })]
```

Comment:
the phrase "add 2 by 3" or "subtract 2 by 3" doesn't read fluently to me, but not a big deal

Reply:
Corpus tests on this release check error message text, and this was easier

Reply:
so the corpus tests have multiplication overflows but not add/sub overflows?

Reply:
I guess so. This is the actual error from the corpus:

```
 assertion `left == right` failed: test /home/runner/work/cedar/cedar/cedar-policy/../cedar-integration-tests/corpus_tests/7b316784cf9e60631768b35cb7a7e15ba6d01c05.json failed for request "Query 0"
  left: Response { decision: Deny, diagnostics: Diagnostics { reason: {}, errors: {"error occurred while evaluating policy `policy0`: integer overflow while attempting to multiply the values `-1537158028109086738` and `60138`"} } }
 right: Response { decision: Deny, diagnostics: Diagnostics { reason: {}, errors: {"error occurred while evaluating policy `policy0`: integer overflow while attempting to multiply -1537158028109086738 by 60138"} } }
```

Reply:
Ah. I didn't realize this changed the error message for add and since there was a conflicting error message change adding backtick around the numbers.

Reply:
Went back and changed the corpus tests since both option meant changing some error message